Orange/preprocess/preprocess.py

```diff
@@ -326,8 +326,7 @@ class Randomize(Preprocess):
     >>> randomizer = Randomize(Randomize.RandomizeClasses)
     >>> randomized_data = randomizer(data)
     """
-    Type = Enum("Randomize",
-                "RandomizeClasses, RandomizeAttributes, RandomizeMetas")
+    Type = 1, 2, 4
```

Comment:
Otherwise, no, it doesn't have to be `Enum`. We're mostly not using it. I just liked it and asked why it was removed.